README.md

```diff
@@ -2,3 +2,20 @@
 A Latex Template for your bachelor / master thesis
 
 Please note: This is only a template. You have to adapt the template to your thesis and discuss the structure of your thesis with your supervisor
+
```

Comment:
Please add a heading for this section - I think it is not immediately obvious that this is only important for existing repositories hosted on GitHub

Reply:
Good point! I inserted some headings and also explained how to disable the workflow. I also mentioned the billing aspects for GH Actions briefly.